### PR TITLE
RDoc-2950 Documentation for Single File Deployment

### PR DESCRIPTION
**Pull request by Danielle9897**
**Related issue:**
https://issues.hibernatingrhinos.com/issue/RDoc-2950/Documentation-for-Single-File-Deployment

### CODE REVIEW

**Comment by arekpalinski** [comment]
@gregolsky or @ppekrol can you please review it?

**Comment by ppekrol** [comment]
@Danielle9897 @reebhub please contact @gregolsky on Monday about this, we need to add few enhancements

File: Documentation/6.1/Raven.Documentation.Pages/migration/server/data-migration.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+{PANEL: Replacing the binaries}
+
+* Starting with RavenDB `6.1`, the `.exe` files are self-contained,  
+  bundling all necessary `.dll` files and dependencies into a single executable.
+
+* When migrating to `6.1`, ensure you **replace ALL** previous binaries with the updated `.exe` files:
+    * _Raven.Server.exe_
+    * _Raven.Debug.exe_
+    * _rvn.exe_
```
Comment (gregolsky) [inline Documentation/6.1/Raven.Documentation.Pages/migration/server/data-migration.markdown:25]:
```suggestion
* When migrating to `6.1`, ensure you first **remove ALL** previous binaries (DLL and EXE files) and add the `.exe` files from the update package:
    * _Raven.Server.exe_
    * _Raven.Debug.exe_
    * _rvn.exe_
```

Comment (Danielle9897) [reply Documentation/6.1/Raven.Documentation.Pages/migration/server/data-migration.markdown:25]: I will create a new PR with this fix for 6.2

**Comment by Danielle9897** [comment]
Closing this PR in favor of [PR #1902](https://github.com/ravendb/docs/pull/1902), which is for **v6.2**.

**Pull request closed without merging**